addons/api/tests/unit/adapters/application-test.js

```diff
@@ -10,6 +9,12 @@ module('Unit | Adapter | application', function (hooks) {
   setupTest(hooks);
   setupMirage(hooks);
 
+  let config;
+
+  hooks.beforeEach(function () {
```

Comment:
Do we want to retrieve the value of `config` before each test? or once before all tests?

Reply:
It can only be retrieved in the context of a test case, so it is looked up every time.